### PR TITLE
Init SIWS example that uses `LitAction.checkConditions` for simple SIWS auth

### PR DESCRIPTION
**Pull request by spacesailor24**
This code example has a similar implementation (#64) that implements SIWS auth for generating session sigs and decrypting a string. This code example simplifies the implementation to just authenticating a SIWS message to allow for an easy how to guide to be written

### CODE REVIEW

File: siws-accs/browser/src/litActionSiws.ts [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    message += `\n\n${fields.join("\n")}`;
+  }
+
+  return message;
+}
+
+async function verifySiwsSignature(
+  message: string,
+  signatureBase58: string,
+  publicKeyBase58: string
```
Comment (awisniew207) [inline siws-accs/browser/src/litActionSiws.ts:42]:
```suggestion
  message,
  signatureBase58,
  publicKeyBase58
```

File: siws-accs/browser/src/litSiws.ts [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    );
+
+    console.log("🔄 Connecting to Lit Node Client...");
+    litNodeClient = new LitNodeClient({
+      litNetwork: LitNetwork.DatilDev,
+      debug: false,
+    });
+    await litNodeClient.connect();
+    console.log("✅ Connected to Lit Node Client");
+
```
Comment (awisniew207) [inline siws-accs/browser/src/litSiws.ts:88]:
```suggestion

console.log("🔄 Attempting SIWS Authentication...");
```

File: siws-accs/browser/src/litSiws.ts [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+    const response = await litNodeClient.executeJs({
+      code: litActionSiws,
+      sessionSigs: await getSessionSigs(litNodeClient, ethersSigner),
+      jsParams: {
+        siwsObject: JSON.stringify(siwsObject),
+        solRpcConditions,
+      },
+    });
+
```
Comment (awisniew207) [inline siws-accs/browser/src/litSiws.ts:97]:
```suggestion

console.log("✅ Successfully authenticated with SIWS message");
```

File: siws-accs/browser/package.json [hunk 4]
```diff
@@ -0,0 +1,51 @@
+{
+  "name": "browser-template",
```
Comment (awisniew207) [inline siws-accs/browser/package.json:2]:
```suggestion
  "name": "siws-accs-browser",
```

**Comment by awisniew207** [comment]
Needs: 
README
Datil-Test capability?

Let me know if you'd want me to implement these

File: siws-accs/browser/src/litSiws.ts [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+import { LIT_RPC, LitNetwork } from "@lit-protocol/constants";
+import { ethers } from "ethers";
+import {
+  createSiweMessage,
+  generateAuthSig,
+  LitActionResource,
+} from "@lit-protocol/auth-helpers";
+
+import { SiwsObject } from "./types";
+import litActionSiws from "./dist/litActionSiws.js?raw";
```
[the remark this answers is not in the document]
Comment (spacesailor24) [reply siws-accs/browser/src/litSiws.ts:12]: Need to run `yarn build:lit-action` to compile the Lit Action before running. The outfile is in the `dist` dir

**Comment by spacesailor24** [comment]
@awisniew207 Added a `README` and update the example to use Datil-test (handles minting Capacity Credit if not provided)

File: siws-accs/browser/README.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+- The code example also expects the [Phantom wallet browser extension](https://chromewebstore.google.com/detail/phantom/bfnaelmomeimhlpmgjnjophhpkkoljpa?hl=en) to be installed and setup with a Solana wallet
+
+## Installation and Setup
+
+1. Clone the repository
+2. `cd` into the code example directory: `cd siws-accs/browser`
+3. Install the dependencies: `yarn`
+4. Create and fill in the `.env` file: `cp .env.example .env`
+   - `VITE_ETHEREUM_PRIVATE_KEY`: **Required** This is the Ethereum private key that will be used to mint a Lit Capacity Credit and create Lit Session Signatures
+5. Build the Lit Action file: `yarn build:lit-action`
```
Comment (awisniew207) [inline siws-accs/browser/README.md:25]: I'd add the installation for deno before this step. I didn't have it on my machine, so I ran into an issue when following the instructions

Comment (awisniew207) [reply siws-accs/browser/README.md:25]: Maybe:

```
Running this example requires the deno JavaScript runtime. You can find installation instructions [here](https://deno.com/)
```

Or something along those lines

**Pull request merged**